src/containerapp/azext_containerapp/_help.py

```diff
@@ -1897,6 +1897,20 @@
               --cpu 0.5 --memory 1Gi --target-port 80 --registry-server myregistry.azurecr.io \\
               --registry-username myregistry --registry-password $REGISTRY_PASSWORD \\
               --location eastasia
+    - name: Create or update a Session Pool with container type CustomContainer and Managed Identity to authenticate Azure container registry
```

Comment:
How a customer can remove existing assigned system or user-assigned identity? we heard this as constant feedback in ACA so can we add example ensuring there is a way to achieve this.

Reply:
Will handle sessionpool update support with MI in next pr